docs/examples/use_cases/index.py

```diff
@@ -0,0 +1,16 @@
+doc(title=("Operations", "="),
+    options=":maxdepth: 2",
+    entries=[
+        # Must use explicit example_entry, for the doc mechanism to not expect README.py file
+        # with index. Only .rst and .ipynb are detected as examples automatically
+        example_entry("video_superres/README"),
+        example_entry("pytorch/resnet50/pytorch-resnet50"),
```

Comment:
```suggestion
        "pytorch/resnet50/pytorch-resnet50.rst",
```
It is the rest file.

Reply:
done